src/Unicorn/Configuration/MicroConfiguration.cs

```diff
@@ -90,9 +90,9 @@ public virtual object Activate(Type type, KeyValuePair<string, object>[] unmappe
 			{
 				var currentParam = ctorParams[parameterIndex];
 
-				if (unmappedConstructorParameters.Any(kv => kv.Key.Equals(currentParam.Name, StringComparison.Ordinal)))
```

Comment:
What necessitated making parameter mapping case insensitive? If not specific to TFS, would prefer that it remain sensitive.